composer.json

```diff
@@ -125,6 +125,7 @@
     "vimeo/psalm": "^4.10"
   },
   "conflict": {
+    "jms/serializer-bundle": "4.1.0 || 4.1.1"
```

Comment:
How do you know that `4.1.1` will be broken, too?

Reply:
not at all, that was reason why i lock version on the first time but you give thumbs up for this approach

Reply:
but honestly, how can we be sure that they will even remove the bug in either of those two versions

Reply:
It won't be removed in `4.1.0` as it's already released, thus we know it's broken.

Yes, we can't be sure it'll be fixed in the next version, but as it's blocking usage in dev mode, there's a high chance it will be resolved.

Reply:
So if bug is still available, we will update `conflict` with new one and so on?

Reply:
@gjorgic yes. But I am certain it will be resolved in 4.1.1

Reply:
ok, i'm asking in general how you will handle if something like that happen, thx